packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -7,6 +7,7 @@ import { ApiClient } from '../.'
  * - 'all' to list the whole DB
  */
 type OwnerType = 'mine' | 'all' | string
+type TokenType = `${string}@${string}`
```

Comment:
this is actually called account in blockchain, so instead of calling it AccountType, properly name it AccountRegex or AccountStr??

Reply:
JJ suggested TokenType. I would like to name the variable ending with Type. So I shall change to AccountRegexType. Good?

Reply:
i'm okay